src/bin/ion/commands/beta/inspect.rs

```diff
@@ -70,39 +69,30 @@ in one or more containers, those containers will be displayed too."
                 .long_help(
                     "When specified, the inspector will stop printing values after
 processing `n` bytes of Ion data. If `n` falls within a value, the
-complete value will be displayed."
-                )
+complete value will be displayed.",
+                ),
         )
 }
 
-// Create a type alias to simplify working with a shared, mutable reference to our output stream.
-type OutputRef = Rc<RefCell<dyn io::Write>>;
+// Create a type alias to simplify working with a shared reference to our output stream.
+type OutputRef = Box<dyn io::Write>;
```

Comment:
Now that we're not using a callback system, there's no need to share the output stream between multiple owners. We can discard the wrapper types that enabled that sharing (`Rc<RefCell<_>>`).